test_proxy.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-# Caddy Test Script (for /apps and /manifest routes)
+# Proxy Test Script (for /apps and /manifest routes)
```

Comment:
FWIW this test is not sufficient for many reasons. For now I've chosen not to replace it to validate that the server is behaving as it was previously, however I'd like to revisit this and add more robust tests.